llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1603,7 +1603,7 @@ static void transformRecipestoEVLRecipes(VPlan &Plan, VPValue &EVL) {
   LLVMContext &Ctx = CanonicalIVType->getContext();
   VPValue *AllOneMask = Plan.getOrAddLiveIn(ConstantInt::getTrue(Ctx));
 
-  for (VPUser *U : Plan.getVF().users()) {
+  for (VPUser *U : SmallVector<VPUser *>(Plan.getVF().users())) {
```

Comment:
```suggestion
  for (VPUser *U : to_vector(Plan.getVF().users())) {
```

Should be possible to use `to_vector` I think